src/main/java/org/gnieh/logback/config/ConfigConfigurator.java

```diff
@@ -248,4 +277,38 @@ private void configureLogger(LoggerContext loggerContext, Map<String, Appender<I
 
 	}
 
+	@SuppressWarnings({ "unchecked" })
+	private Object configureObject(LoggerContext loggerContext, Map<String, Object> params) throws ReflectiveOperationException {
```

Comment:
I’d like it more if this method takes as parameter the expected type when used to configured logging elements (e.g. when we are expecting an `Appender`, which requires a `class` key with the fully qualified class name). In other cases, the type may be raw `Config` objects.